string filename
bool overwrite

---

bool status